{"uuid":"3f7d0a75-cb00-4ee6-9e3a-6897626d7398","name":"Scenario Outline: The user is presented with error message for invalid code in tryEditor for Max Consecutive Ones","children":["75f58a77-997b-4b2e-a063-64b281fc859b"],"befores":[],"afters":[{"name":"Hooks.Hooks.a(io.cucumber.java.Scenario)","status":"passed","stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1696883507929,"stop":1696883507930},{"name":"Hooks.Hooks.a(io.cucumber.java.Scenario)","status":"passed","stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1696883508012,"stop":1696883508012},{"name":"Hooks.Hooks.a(io.cucumber.java.Scenario)","status":"passed","stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1696883508070,"stop":1696883508070},{"name":"Hooks.Hooks.a(io.cucumber.java.Scenario)","status":"passed","stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1696883510111,"stop":1696883510112}],"start":1696883507554,"stop":1696883510112}